explore-analyze/query-filter/languages/esql-query-log.md

```diff
@@ -0,0 +1,137 @@
+---
+applies_to:
+  stack: ga
+  serverless: ga
+navigation_title: "Query log"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/esql-query-log.html
+---
+
+
+
+# ES|QL query log [esql-query-log]
+
+
+Query Log allows to log ES|QL queries based on their execution time. 
+You can use these logs to investigate, analyze or troubleshoot your cluster’s historical ES|QL performance.
+
+ES|QL query log reports task duration at coordinator level, but might not encompass the full task execution time observed on the client. For example, logs don’t surface HTTP network delays.
+
+Events that meet the specified threshold are emitted into [{{es}} logging](../../../deploy-manage/monitor/logging-configuration/update-elasticsearch-logging-levels.md). 
+
+These logs can be found in local {{es}} service logs directory. Slow log files have a suffix of `_esql_querylog.json`.
```

Comment:
This will need to be updated when we have the Beats connector and stack monitoring enabled.